tools/run_tests/run_interop_tests.py

```diff
@@ -661,9 +661,6 @@ def build_interop_image_jobspec(language, tag=None):
           
     if args.http2_interop:
       for test_case in _HTTP2_TEST_CASES:
-        if server_name == "go":
```

Comment:
Looks like this change has introduces these failures:
tests.cloud_to_cloud:http2:go_server:tls
tests.cloud_to_cloud:http2:go_server:framing